### PR TITLE
Music: update instructions styling

### PR DESCRIPTION
**Pull request by breville**
This makes some adjustments to the Music Lab instructions' styling and layout, now that we are seeing some longer Markdown instructions being produced.  The  Markdown content's styling is much tighter, the feedback message and Next button are always visible, and the Next button gets new styling too.

#### With some Markdown instructions

Screenshot 2023-06-22 at 10 02 38 PM

#### With a feedback message

Screenshot 2023-06-22 at 10 02 43 PM

#### With the Next button

Screenshot 2023-06-22 at 10 02 54 PM

### CODE REVIEW

File: apps/src/music/views/Instructions.jsx [hunk 1]
```diff
@@ -37,11 +36,7 @@ const Instructions = ({
     analyticsReporter.onInstructionsVisited(currentPanel + 1);
   }, [currentPanel, analyticsReporter]);
 
-  const nextPanel = getNextPanel();
-
-  const progressText = progressionStep
-    ? `${currentPanel + 1}/${levelCount}`
-    : '';
+  const isNextPanel = progressState.satisfied ? !!getNextPanel() : false;
```
Comment (sanchitmalhotra126) [inline apps/src/music/views/Instructions.jsx]: nit: maybe `hasNextPanel`?

File: apps/src/music/views/Instructions.jsx [hunk 2]
```diff
@@ -156,17 +133,37 @@ const InstructionsPanel = ({
...
+            />
+          </div>
+          {onNextPanel && (
+            <button
+              id="instructions-feedback-button"
+              type="button"
+              onClick={() => onNextPanel()}
+              className={moduleStyles.buttonNext}
+            >
+              Next
```
Comment (sanchitmalhotra126) [inline apps/src/music/views/Instructions.jsx]: Localized string for this?

File: apps/src/music/views/instructions.module.scss [hunk 3]
```diff
@@ -81,51 +82,50 @@
...
+      color:  $neutral_light;
+    }
 
-      &.textVertical {
-        padding: 0 10px;
-        margin: 10px 0;
+    .markdownText {
+      line-height: 0;
+
+      h1, h2 {
```
Comment (sanchitmalhotra126) [inline apps/src/music/views/instructions.module.scss:94]: I'm guessing we need this for the markdown styling? Can't use classnames instead?

**Review by sanchitmalhotra126 — APPROVED** [review]

File: apps/src/music/views/Instructions.jsx (hunk 1 above)
Comment (breville) [reply apps/src/music/views/Instructions.jsx]: Good call.

File: apps/src/music/views/Instructions.jsx (hunk 2 above)
Comment (breville) [reply apps/src/music/views/Instructions.jsx]: Good call.

File: apps/src/music/views/instructions.module.scss (hunk 3 above)
Comment (breville) [reply apps/src/music/views/instructions.module.scss:94]: Unfortunately this is all we get from the `#` and `##` syntax being used.

**Pull request merged**